bzl/macros/pytest_wrapper.py

```diff
@@ -0,0 +1,6 @@
+import sys
+
+import pytest
+
+if __name__ == "__main__":
+    sys.exit(pytest.main(sys.argv[1:]))
```

Comment:
you may want to use the env vars `TEST_TMPDIR` as well as `TESTBRIDGE_TEST_ONLY`

see https://github.com/jessecureton/python_bazel_template for some neat suggestions

Reply:
very neat, great find. I copied that into my next branch but fixed up the arg paths